pikaday.js

```diff
@@ -1009,6 +1014,8 @@
             for (var c = 0; c < opts.numberOfMonths; c++) {
                 html += '<div class="pika-lendar">' + renderTitle(this, c, this.calendars[c].year, this.calendars[c].month, this.calendars[0].year, randId) + this.render(this.calendars[c].year, this.calendars[c].month, randId) + '</div>';
             }
+            
+            html+=renderTodayLink();
```

Comment:
There should be an option to enable this. With default to false for backward compatibility.